presto-phoenix/src/main/java/io/prestosql/plugin/phoenix/PhoenixClient.java

```diff
@@ -13,6 +13,8 @@
  */
```

Comment:
> Minor cleanup in Phoenix connector

What kind of cleanup? Commit message is ambiguous. Can you please be more specific.

Reply:
Now we pass the table properties via `JdbcClient` (`PhoenixClient`)